src/docs/product/discover-queries/query-builder.mdx

```diff
@@ -85,20 +85,22 @@ When the "Display" selection is either "Top Period" or "Top Daily", an optional
 
 When your "Display" selection is either "Total Period", "Previous Period", "Total Daily" or "Bar Chart", you can set up to three y-axes by selecting multiple checkboxes in the "Y-Axis" dropdown.
 
-## Filter by Table Columns
+## Group & Stack With Table Columns {#filter-by-table-columns}
```

Comment:
@doralchan - I think this aligns with our conversation and the existing description, however, I'm a bit concerned because in the columns modal text, it uses the word "group" in the way we're using "stack" here. I think we should probably decide on exactly which terminology we want to use and then be consistent between docs and the UI.